proc/process.c

```diff
@@ -1365,7 +1367,7 @@ static int process_execve(thread_t *current)
 	else {
 		/* Reinitialize process */
 		map = current->process->mapp;
-		proc_changeMap(current->process, NULL, NULL);
+		proc_changeMap(current->process, NULL, NULL, NULL);
```

Comment:
setting process->imapp here changes behaviour for if in line 1375

Reply:
Fixed, thanks